web-components/src/components/link/Link.ts

```diff
@@ -26,7 +37,7 @@ export namespace Link {
   export class ELEMENT extends LitElement {
     @property({ type: String, attribute: false }) color: Color = "";
     @property({ type: String }) ariaLabel = "";
-    @property({ type: String }) role: Role = "link";
+    @property({ type: String }) ariaRole: Role = "link";
```

Comment:
also make this as attribute e.g aria-role:

    @property({ type: string, reflect: true, attribute: "aria-role" }) ariaRole = "link";

Reply:
aria-role gives lint issues. Hence using as ariaRole only like ariaLabel.
![Screenshot 2024-03-05 at 10 52 22 AM](https://github.com/momentum-design/momentum-ui/assets/41052645/0f7063f9-6b6f-4882-adad-d1d5d6a2030a)